contracts/p0/AssetRegistry.sol

```diff
@@ -12,6 +12,8 @@ contract AssetRegistryP0 is ComponentP0, IAssetRegistry {
     using FixLib for uint192;
     using EnumerableSet for EnumerableSet.AddressSet;
 
+    uint256 public constant GAS_TO_RESERVE = 900000; // just enough to disable basket on n=128
```

Comment:
Yes, but it would go un-used. Not sure how annoying this is for people...we could just disable the basket everytime without checking `quantity()` too.

Reply:
I think it might be ok though. But worth confirming all together.